cmd/otelcol/Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM alpine:3.9 as certs
 RUN apk --update add ca-certificates
+RUN mkdir -m 777 /scratchtmp
```

Comment:
It might be easier to define `/tmp` as a volume:

```
VOLUME ["/tmp"]
```

It would also help people who would want to have this available elsewhere.